css/css-grid/grid-model/grid-button-001.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<title>Grid items work inside a button</title>
+<meta name="assert" content="When a button is set to display: grid, its children should flow into its grid cells">
+<link rel="author" title="Bryan Robinson" href="bryanlrobinson@gmail.com">
+<link rel="help" href="https://drafts.csswg.org/css-grid-1/#grid-containers">
+<style>
+  .grid { display: grid; grid-template-columns: 100px 200px; border: 2px solid red; box-sizing: border-box; /* for visual */}
```

Comment:
Please avoid using `red` if it's visible when the test runs properly (it can be confusing). Better use `black` or any other color here.
Nit: Also the comments `/* for visual */` are not really needed, you can remove them.

Reply:
Makes sense! I wasn't thinking